docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -479,7 +485,7 @@ Write operations are supported for tables stored on the following systems:
 
   Writes to :doc:`Amazon S3 <hive-s3>` and S3-compatible storage must be enabled
   with the ``delta.enable-non-concurrent-writes`` property. Writes to S3 can
-  safely be made from multiple Trino clusters, however write collisions are not
+  safely be made from multiple Trino clusters; however, write collisions are not
```

Comment:
![image](https://user-images.githubusercontent.com/3756245/230651284-9d6743c7-bdbb-407a-96c1-a1ae71b2ddd7.png)

Reply:
No need for such references.